addons/account_fleet/models/fleet_vehicle_log_services.py

```diff
@@ -0,0 +1,36 @@
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+class FleetVehicleLogServices(models.Model):
+    _inherit = 'fleet.vehicle.log.services'
+
+    account_move_line = fields.Many2one('account.move.line')
+    account_move_state = fields.Selection(related="account_move_line.parent_state", store=True)
+    account_move = fields.Many2one('account.move', related="account_move_line.move_id", store=True)
+    amount = fields.Monetary('Cost', compute="_compute_amount", readonly=False, store=True, tracking=True)
```

Comment:
Why stored ? Also, this is a related field in that case, but you miss the inverse right (could lead to desynchronization between the account move line) ? 
```suggestion
    amount = fields.Monetary('Cost', related="account_move_line.price_subtotal", tracking=True)
```